pkg/acceptance/cluster/dockercluster.go

```diff
@@ -70,18 +69,10 @@ var waitOnStop = flag.Bool("w", false, "wait for the user to interrupt before te
 var maxRangeBytes = *zonepb.DefaultZoneConfig().RangeMaxBytes
 
 // CockroachBinary is the path to the host-side binary to use.
-var CockroachBinary = flag.String("b", func() string {
-	rootPkg, err := build.Import("github.com/cockroachdb/cockroach", "", build.FindOnly)
-	if err != nil {
-		panic(err)
-	}
-	// NB: This is the binary produced by our linux-gnu build target. Changes
-	// to the Makefile must be reflected here.
-	return filepath.Join(rootPkg.Dir, "cockroach-linux-2.6.32-gnu-amd64")
-}(), "the host-side binary to run")
+var CockroachBinary = flag.String("b", "", "the host-side binary to run")
 
-func exists(path string) bool {
-	if _, err := os.Stat(path); oserror.IsNotExist(err) {
+func fileExists(path string) bool {
+	if f, err := os.Stat(path); oserror.IsNotExist(err) || f.IsDir() {
```

Comment:
I imagine it is rather unlikely given the environment that we run this command in, but if err is non-nil but not an IsNotExist error, then `f` will likely be nil.

Reply:
Oh, right... Maybe I should have left it as is. Just wanted to guard against the path being a directory, but meh. I'll revert this hunk.